src/test/java/jenkins/plugin/mockloadbuilder/WithMockLoadStepTest.java

```diff
@@ -24,39 +24,43 @@
 
 package jenkins.plugin.mockloadbuilder;
 
+import static org.junit.Assume.assumeFalse;
+
 import hudson.Functions;
-import org.junit.ClassRule;
-import org.junit.Test;
 import org.jenkinsci.plugins.workflow.cps.CpsFlowDefinition;
 import org.jenkinsci.plugins.workflow.job.WorkflowJob;
-import static org.junit.Assume.assumeFalse;
+import org.junit.ClassRule;
 import org.junit.Rule;
+import org.junit.Test;
 import org.jvnet.hudson.test.BuildWatcher;
 import org.jvnet.hudson.test.JenkinsSessionRule;
 
 public class WithMockLoadStepTest {
 
-    @ClassRule public static BuildWatcher buildWatcher = new BuildWatcher();
+    @ClassRule
+    public static BuildWatcher buildWatcher = new BuildWatcher();
 
-    @Rule public JenkinsSessionRule rr = new JenkinsSessionRule();
+    @Rule
+    public JenkinsSessionRule rr = new JenkinsSessionRule();
```

Comment:
Sadly loses the semantic distinction between attributive and determinative annotations.